src/components/Select/common/InternalSelect.tsx

```diff
@@ -306,7 +306,7 @@ export const InternalSelect = ({
     e.preventDefault();
     e.stopPropagation();
     if (allowCreateOption) {
-      onSelect(search, "custom");
+      onSelect(search, "custom", e);
```

Comment:
I didn't approve right away because I wanted to check something, but then I got distracted, sorry.

The thing I wanted to check is this. There is another place where onSelect gets called:
https://github.com/ClickHouse/click-ui/blob/main/src/components/Select/common/InternalSelect.tsx#L552

the same goes for multi select items, I think?
https://github.com/ClickHouse/click-ui/blob/main/src/components/Select/common/InternalSelect.tsx#L634

they get there through the context, and it is too easy to miss, so wanted to point that out

Reply:
Done